src/components/DisplaySideArticles.jsx

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import { connect } from "react-redux";
+import { getArticles } from "../modules/getArticlesData";
+
+const DisplaySideArticles = props => {
+  const getArticleShowData = async () => {
+    const articlesData = await getArticles();
+    props.changeSideArticlesData(articlesData);
+  };
+
+  if (!props.sideArticles) {
+    getArticleShowData();
+  }
+
+  if (props.sideArticles && props.sideArticles.articles.length > 0) {
+    props.changeCurrentArticleId(props.sideArticles.articles[0].id);
+  }
+
+  let articlesList;
+
+  if (props.sideArticles && props.sideArticles.articles.length > 0) {
+    let articlesList = props.sideArticles.articles.map(x => {
```

Comment:
change `x` to something more descriptive

Reply:
yes